### ISSUE TITLE
Pass endPoint as second argument of onSelectionChange

### ISSUE THREAD

**Issue by orangecoloured** [aurbano/react-ds#46]
Having the `endPoint` as second argument of `onSelectionChange` could help with some UX stuff, like popping up some actions panel for highlighted elements in that point. Atm an additional listener is required to accomplish this.

**Comment by aurbano** [comment]
@orangecoloured makes sense, I'm a bit busy these days but feel free to send a PR with the change if you want! :)

**Comment by orangecoloured** [comment]
https://github.com/aurbano/react-ds/pull/48
